packages/template-retail-react-app/app/components/store-locator/index.jsx

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import React from 'react'
+import PropTypes from 'prop-types'
+import {
+    Modal,
+    ModalBody,
+    ModalCloseButton,
+    ModalContent,
+    useBreakpointValue
+} from '@salesforce/retail-react-app/app/components/shared/ui'
+import StoreLocatorContent from '@salesforce/retail-react-app/app/components/store-locator/store-locator-content'
+
+const StoreLocatorModal = (props) => {
+    const {isOpen, setIsOpen} = props
+
+    const isDesktopView = useBreakpointValue({base: false, lg: true})
+
+    return (
+        <>
+            {isDesktopView ? (
```

Comment:
Using a breakpoint value boolean instead of the `<HideFrom...>` components because the latter wasn't super compatible for chakra UI modals: 
1. Modal overrides some aspect of display, requiring inline styling anyways
2. display "none" applied to modal still rendered in the DOM